torch/csrc/jit/backends/backend_detail.cpp

```diff
@@ -140,13 +150,27 @@ Module codegen_backend_module(
   // backend class.
   static const auto create_backend_ct = CodeTemplate(R"(
             def __create_backend(self):
-                self.__backend = $name()
+                backend = self.__backend
+                if (backend is None):
+                    self.__backend = $name()
             )");
   TemplateEnv create_backend_te;
   create_backend_te.s("name", qual_backend_name.qualifiedName());
   loweredModule.define(
       create_backend_ct.format(create_backend_te), loweredModuleResolver());
 
+  // Helper function to expose backend.is_available() to Module generation code.
+  loweredModule.define(
+      R"(
+            def __is_available(self):
+                self.__create_backend()
+                backend = self.__backend
```

Comment:
This copy may look confusing for readers out of TS. Could we add some comment here

Reply:
Not used anymore.